lib/Rpc/RpcServer.cpp

```diff
@@ -1123,6 +1135,90 @@ bool RpcServer::on_get_transactions(
     return true;
 }
 
+bool RpcServer::onGetTransactionsByHeights(
+        const COMMAND_RPC_GET_TRANSACTIONS_BY_HEIGHTS::request &req,
+        COMMAND_RPC_GET_TRANSACTIONS_BY_HEIGHTS::response &res)
+{
+    try {
+        std::vector<Crypto::Hash> vh;
+
+        size_t additor = 100; // possible to increase
+        for (size_t i = req.startBlock; i <= req.startBlock + additor; i++) {
+            Block blk;
+            uint32_t h = static_cast<uint32_t>(i);
+            Crypto::Hash blockHash = m_core.getBlockIdByHeight(i);
+            if (blockHash == NULL_HASH) {
+                throw JsonRpc::JsonRpcError{
+                        CORE_RPC_ERROR_CODE_TOO_BIG_HEIGHT,
+                        std::string("To big height: ")
```

Comment:
I think you mean "Too big height" really.
```suggestion
                        std::string("Too big height: ")
```